scripts/ci/run-test-make-install.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash -e
+
+${TESTCMD} --label=make-install --timeout=30m make -w install
+echo "class H { static void Main () { System.Console.WriteLine (\"Hello World: \" + System.DateTime.Now); } }" > ${MONO_REPO_ROOT}/tmp/hello.cs
+
+MONO_PREFIX=${MONO_REPO_ROOT}/tmp/monoprefix
+export DYLD_FALLBACK_LIBRARY_PATH=$MONO_PREFIX/lib:$DYLD_LIBRARY_FALLBACK_PATH
+export LD_LIBRARY_PATH=$MONO_PREFIX/lib:$LD_LIBRARY_PATH
+export PATH=$MONO_PREFIX/bin:$PATH
+
+${TESTCMD} --label=check-prefix-mcs --timeout=1m mcs /out:${MONO_REPO_ROOT}/tmp/hello.exe ${MONO_REPO_ROOT}/tmp/hello.cs 
+${TESTCMD} --label=check-prefix-roslyn --timeout=1m csc /out:${MONO_REPO_ROOT}/tmp/hello.exe ${MONO_REPO_ROOT}/tmp/hello.cs
+${TESTCMD} --label=check-prefix-aot --timeout=1m mono --aot ${MONO_REPO_ROOT}/tmp/hello.exe
+${TESTCMD} --label=check-prefix-llvmaot --timeout=1m mono --aot=llvm,llvm-path=/usr/lib/mono/llvm/bin ${MONO_REPO_ROOT}/tmp/hello.exe
+${TESTCMD} --label=check-prefix-llvmjit --timeout=1m mono --llvm ${MONO_REPO_ROOT}/tmp/hello.exe
```

Comment:
Could we add also `msbuild /help` or something like that?

Reply:
I think testing msbuild via /help won't achieve anything, it'd need to compile a .cs file. It's a good idea as a baseline to make sure we at least have the dll i guess

Reply:
This PR doesn't address msbuild. A future change can add "build, install, and use msbuild" as a step.